src/core.jl

```diff
@@ -127,7 +132,7 @@ function send_value!(node::Signal, x, timestep)
     # Set the value and do actions
     node.value = x
     for action in node.actions
-        do_action(action, timestep)
+        action_queue[action.recipient.value] = action
```

Comment:
Having said that, it seems to be happening on these builds: https://ci.appveyor.com/project/shashi/reactive-jl/build/1.0.4/job/o44rh18pio50ok8r
https://ci.appveyor.com/project/shashi/reactive-jl/build/1.0.4/job/63i6ry0w3fgbxm94

😃

Not sure how...